crates/websocket-proxy/src/registry.rs

```diff
@@ -30,6 +30,7 @@ impl Registry {
                         Ok(_) => {
                             trace!(message = "message sent to client", client = client.id());
                             metrics.sent_messages.increment(1);
+                            metrics.bytes_broadcasted.increment(msg.len() as u64);
```

Comment:
wondering if its useful here to have an attribute by client (assuming low cardinality of client ids)?

Reply:
I would expect there to be somewhere in the neighborhood of low thousands of clients (depending on the chain). 

I would probably disable indexing the label on our infra (as this endpoint is public and any number of clients can join), but if it would be useful to ya'll happy to add